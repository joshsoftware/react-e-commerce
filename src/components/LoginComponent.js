import React from 'react';
import FormCard from './FormCard';
import ContainerWrapper from './ContainerWrapper';
import RowWrapper from './RowWrapper';
import ColumnWrapper from './ColumnWrapper';

const LoginComponent = () => {
  return (
    <ContainerWrapper
      styleClass={'container pt-3 text-center h-100 '}
      data={
        <RowWrapper
          data={<ColumnWrapper sm={{ size: 6, offset: 3 }} data={<FormCard type={'login'} />} />}
        />
      }
      fluid={true}
    />
  );
};

export default LoginComponent;
