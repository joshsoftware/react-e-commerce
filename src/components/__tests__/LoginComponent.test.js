import { render, cleanup } from '@testing-library/react';
import React from 'react';
import LoginComponent from '../LoginComponent';

afterEach(cleanup);

describe('LoginComponent', () => {
  it('Must return LoginComponent', () => {
    const { asFragment } = render(<LoginComponent />);
    expect(asFragment()).toMatchSnapshot();
  });
});
