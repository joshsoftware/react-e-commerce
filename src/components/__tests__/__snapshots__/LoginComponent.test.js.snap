// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginComponent Must return LoginComponent 1`] = `
<DocumentFragment>
  <div
    class="container pt-3 text-center h-100  container-fluid"
  >
    <div
      class="row"
    >
      <div
        class="col-sm-6 offset-sm-3"
      >
        <div
          class="card"
        >
          <div
            class="card-body"
          >
            <h3>
              Login
            </h3>
            <hr />
            <form
              class=""
            >
              <div
                class="form-group"
              >
                <label
                  class=""
                  for="exampleEmail"
                >
                  Email
                </label>
                <input
                  class="form-control"
                  name="email"
                  placeholder="example@company.com"
                  type="email"
                  value=""
                />
                <div
                  class="invalid-feedback"
                />
              </div>
              <div
                class="form-group"
              >
                <label
                  class=""
                  for="examplePassword"
                >
                  Password
                </label>
                <input
                  class="form-control"
                  name="password"
                  placeholder="password"
                  type="password"
                  value=""
                />
                <div
                  class="invalid-feedback"
                />
              </div>
              <button
                class="btn btn-secondary"
              >
                Login
              </button>
              <hr />
              <button
                class="btn btn-secondary"
              >
                Sign In With Google
              </button>
            </form>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
